### PR TITLE
RedHat compatibility

### PR DESCRIPTION
**Pull request by fjmulero**
I have done these changes to be able to deploy on CentOS 7 generic machines. Them could be interesting for other developers.

Thanks

### CODE REVIEW

File: templates/etc/systemd/system/kubelet.service.j2 [hunk 1]
```diff
@@ -35,6 +35,7 @@ ExecStart={{k8s_bin_dir}}/kubelet \
   --cni-conf-dir={{k8s_cni_conf_dir}} \
   --cni-bin-dir={{k8s_cni_bin_dir}} \
   --serialize-image-pulls={{k8s_worker_kubelet_serialize_image_pulls}} \
+  --fail-swap-on=false \
```
Comment (githubixx) [inline templates/etc/systemd/system/kubelet.service.j2]: Not sure about this option. There is a lengthy discussion here https://github.com/kubernetes/kubernetes/issues/7294  about swap and why to disable it by default. Can you please make this a variable and add it `defaults/main.yml` defaulting to `false`? This way everyone could override the setting to his needs.

File: tasks/main.yml [hunk 2]
```diff
@@ -51,6 +51,23 @@
...
+    - ebtables
+    - ethtool
+    - ipset
+    - conntrack
+    - iptables
+    - iptstate
+    - net-tools
+  tags:
+    - k8s-worker
+  when: ansible_os_family == "RedHat"
```
Comment (githubixx) [inline tasks/main.yml]: Normally I would like to support only Ubuntu LTS releases but if this is the only change needed to work on CentOS 7 I'll accept it.

Comment (fjmulero) [reply tasks/main.yml]: Thanks

File: templates/etc/systemd/system/kubelet.service.j2 (hunk 1 above)
Comment (fjmulero) [reply templates/etc/systemd/system/kubelet.service.j2]: Ok, no problem

**Pull request merged**